app/LiveEvent.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class LiveEvent extends Model {
+    protected $table = 'live_event';
+
+    public static function getAnnouncement(): object {
```

Comment:
Instead of `object` I think we should be able to define the return to be `LiveEvent`. I'm not sure though - PHP's typing is kind of weird

Reply:
You're right - I forgot that's a think now with return typing...